src/Templates/SearchResultsProductTemplate.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Templates;
+
+/**
+ * ProductSearchTemplate class.
+ *
+ * @internal
+ */
+class SearchResultsProductTemplate {
+
+	const SLUG        = 'search-results-product';
+	const TITLE       = 'Product Search Results';
+	const DESCRIPTION = 'Template used to display search results for products';
```

Comment:
Nit: missing dot in the description.
```suggestion
	const DESCRIPTION = 'Template used to display search results for products.';
```

Reply:
Sidenote: it would be nice to add descriptions to all other WC templates, not in this PR, but maybe a follow-up good first issue?

Reply:
Good point, created a new issue https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6233